bin/resty

```diff
@@ -768,10 +768,7 @@ $main_include_directives
 
 stream {
     access_log off;
-    lua_socket_log_errors off;
```

Comment:
why do we need this change?

Reply:
The lua_socket_log_errors/lua_regex_cache_max_entries directive is not allowed here.
see: 
https://github.com/openresty/lua-nginx-module#lua_socket_log_errors
https://github.com/openresty/lua-nginx-module#lua_regex_cache_max_entries